content/applications/hr/fleet/service.rst

```diff
@@ -6,72 +6,282 @@ To properly maintain a fleet of vehicles, regular maintenance as well as periodi
 needed. Scheduling repairs and managing services for an entire fleet is necessary to ensure all
 vehicles are in good working order when they are needed.
 
+.. _service/service-form:
+
 Create a service record
 =======================
 
-To log a service, go to the main services dashboard by navigating to :menuselection:`Fleet app -->
-Fleet --> Services`. Click :guilabel:`Create` and a service form appears. Fill in the information on
-the form and click :guilabel:`Save`. The only two fields that are required to be populated are
+To log a service for a veicle, go to the main services dashboard by navigating to
+:menuselection:`Fleet app --> Fleet --> Services`. Click the :guilabel:`New` button in the top-left
+corner, and a new service form loads.
+
+Fill out the information on the form. The only two fields that are required to be populated are
 :guilabel:`Service Type` and :guilabel:`Vehicle`.
 
-The fields are:
+The service form automatically saves as data is entered. However, the form can be saved manually at
+any time by clicking the :guilabel:`Save manually` option, represented by a :guilabel:`cloud upload`
+icon.
```

Comment:
Since this function isn't unique to Fleet, but rather all forms in Odoo, I'd personally put this in a note admonition or not include this at all. Up to you though!

Reply:
I see what you mean, but the only reason I do want to keep it, is that it was just a few versions ago that you had to save everything. So if a reader is following this form had a previous version, I think it's good to know. Because even now, there's a few parts or forms here and there you have to manually save (not necessarily in Fleet, but in general across Odoo) while a lot gets auto-saved. I think I will remove these instructions when the last 3 versions are all auto-saving (the versions we support) but do want to include it for the time being.

Reply:
I agree with FEKU's suggestion